src/scripts/merge.js

```diff
@@ -1,19 +1,31 @@
-import path from 'path'
 import { exec } from 'child_process'
-
-import { each } from 'lodash'
+import path from 'path'
 
 if (process.argv.length < 4) {
   throw new Error(
     'Invalid arguments, expected: `node i18n/scripts/merge.js pot_file po_file ...`'
   )
 }
 
-const templatePath = path.resolve(process.cwd(), process.argv[2])
+const [, , ...cliArguments] = process.argv
+
+const flags = []
+const files = []
+
+cliArguments.forEach((arg) => {
+  if (arg.startsWith('-')) {
+    flags.push(arg)
+  } else {
+    files.push(arg)
+  }
+})
+
+const [potFile, ...poFiles] = files
+const templatePath = path.resolve(process.cwd(), potFile)
 
-each(process.argv.slice(3), fileName => {
+poFiles.forEach((fileName) => {
   exec(
-    `msgmerge -o ${fileName} ${fileName} ${templatePath}`,
+    `msgmerge ${flags.join(' ')} -o ${fileName} ${fileName} ${templatePath}`,
```

Comment:
Looks good to me :) 
https://www.gnu.org/software/gettext/manual/html_node/msgmerge-Invocation.html